packages/cdk8s-plus-17/src/service.ts

```diff
@@ -244,12 +259,16 @@ export class Service extends Resource {
       });
     }
 
-    return {
+    return this.type !== ServiceType.EXTERNAL_NAME ? {
```

Comment:
As compared to https://github.com/awslabs/cdk8s/pull/424, I tried to ensure that the spec returned only has the details really needed.

Reply:
I actually like this approach better :)

Reply:
@abierbaum I see that @pusherman PR contains a few additional stuff i'd like to also merge, so lets to merge that one instead? 

@pusherman are you waiting for another review on that PR?

Reply:
@iliapolo Sounds good to me.  Looks like @pusherman liked this one piece that seemed like the only improvement I had.  If he wants to role that into his PR that is great but I am also fine without it.  Just need something to use. :)

Reply:
Yeap, lets give @@pusherman a day or two to respond :)

Reply:
just updated my PR to include the code to only return the needed props needed for an ExternalService.  It's ready to review whenever it passes the build/test checks.  Thanks @iliapolo and @abierbaum!